source/clear-linux/tutorials/multi-boot/multi-boot-mint.rst

```diff
@@ -0,0 +1,145 @@
+.. _multi-boot-mint:
+
+Install Mint\* 18.1 “Serena” MATE
+*********************************
+
+#. Start the Mint installer and follow the prompts.
+
+#. At the :guilabel:`Installation type` screen, choose
+   :guilabel:`Something else`. See Figure 1.
+
+   .. figure:: figures/multi-boot-mint-1.png
+
+      Figure 1: Mint: Installation type
+
+#. Create a new root partition.
+
+   #. Under the :guilabel:`Device` column, select :guilabel:`free space`. See
+      Figure 2.
+
+      .. figure:: figures/multi-boot-mint-2.png
+
+         Figure 2: Mint: Add partition
+
+   #. Click the :guilabel:`+` button.
+
+   #. In the :guilabel:`Size` field, enter a value for the new partition
+      size. For this example, we used *40000 MB*, as shown in Figure 3.
+
+      .. figure:: figures/multi-boot-mint-3.png
+
+         Figure 3: Mint: Configure new partition settings
+
+   #. Set :guilabel:`Use as` to :guilabel:`Ext4 journaling file system`.
+
+   #. Set the :guilabel:`Mount point` to :guilabel:`/`.
+
+   #. Click :guilabel:`OK`.
+
+#. Share the same swap partition created by |CL| with the following
+   steps.
+
+   #. Under :guilabel:`Device` column, select :file:`/dev/sda2`.
+
+   #. Click :guilabel:`Change`.
+
+   #. Confirm :guilabel:`Use as` is set to :guilabel:`Swap area`. See Figure 4.
+
+      .. figure:: figures/multi-boot-mint-4.png
+
+         Figure 4: Mint: Set swap partition
+
+#. Follow the remaining prompts to complete the installation of Mint.
+
+#. At this point, the ability to boot |CL| is lost because `Grub`
+   is the default boot loader. Follow these steps to make the |CL|
+   Systemd-Boot the default boot loader and add Mint as a boot option.
+
+   #. Boot into Mint.
+
+   #. Log in.
+
+   #. Get root permissions.
+
+      .. code-block:: console
+
+         $ sudo -s
```

Comment:
Please remove "Get root permissions." section.

Reply:
Text edits deferred to next phase of updates.